instrumentation/hystrix-1.4/src/test/groovy/HystrixObservableChainTest.groovy

```diff
@@ -19,13 +19,17 @@ import static io.opentelemetry.auto.test.utils.TraceUtils.runUnderTrace
 
 import com.netflix.hystrix.HystrixObservableCommand
 import io.opentelemetry.auto.test.AgentTestRunner
+import io.opentelemetry.auto.test.utils.ConfigUtils
 import rx.Observable
 import rx.schedulers.Schedulers
 
 class HystrixObservableChainTest extends AgentTestRunner {
   static {
     // Disable so failure testing below doesn't inadvertently change the behavior.
     System.setProperty("hystrix.command.default.circuitBreaker.enabled", "false")
```

Comment:
I think both properties should be inside `ConfigUtils.updateConfig`

Reply:
`hystrix.command.default.circuitBreaker.enabled` is configuration for the library under test, not for the instrumentation under test, so not sure it should be under `updateConfig`